### ISSUE TITLE
Implement theme: Royale

### ISSUE THREAD

**Issue by rozniak** [rozniak/xfce-winxp-tc#10]
Issue to track implementation progress for the Royale theme.

**Comment by rozniak** [comment]
Located under `/themes/symphony/royale` in `feat-themes`

**Comment by unexp** [comment]
Been working on it for a bit, but I'm not sure what to name the theme... I think the official theme lists the base theme as Media Center style, but I think either some versions of Royale/Royale Noir (probably the latter) list the base theme as Royale.

Do you have any preference on this?

**Comment by rozniak** [comment]
Please go with 'Media Center style', I think the variant is Royale isn't it?

**Comment by unexp** [comment]
> Please go with 'Media Center style', I think the variant is Royale isn't it?

Yeah, the variant is Royale. Should Royale Noir also be under Media Center style in the future?

**Comment by rozniak** [comment]
I believe so yeah

**Comment by rozniak** [comment]
I am working on porting Royale, probably the remaining themes because they're the oldest tasks that still aren't done.
